src/styles/index.scss

```diff
@@ -1,3 +1,166 @@
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+  --white: #fff;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
+  background-color: #f4f4f4;
+}
+
+.product {
+  display: flex;
+  justify-content: center;
+
+  &-image-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &-card {
+    display: flex;
+    flex-direction: column;
+    width: 200px;
+    background-color: var(--white);
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
+  }
+
+  &-image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &-title {
+    color: var(--main-accent);
+    font-family: "Roboto", sans-serif;
+    font-size: 12px;
+
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 0 4px;
+  }
+
+  &-code {
+    color: var(--secondary);
+    font-family: "Roboto", sans-serif;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &-rate-quantity {
+    display: flex;
+    height: 16px;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  &-rate-quantity-text {
+    color: var(--main-accent);
+    text-align: right;
+    font-family: "Roboto", sans-serif;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &-price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+    align-items: flex-end;
+  }
+
+  &-price-text {
+    color: var(--secondary, #616070);
+    font-family: "Roboto", sans-serif;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &-price-number {
+    color: var(--main-accent, #060b35);
+    text-align: right;
+    font-family: "Roboto", sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    margin-top: 16px;
+    flex-shrink: 0;
+    background: var(--blue-accent);
+    border-radius: 5px;
+    border: 1px solid var(--blue-accent, #00acdc);
+    cursor: pointer;
+
+    &-link {
+      color: var(--white, #00acdc);
+      text-align: center;
+      font-family: "Roboto", sans-serif;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      text-transform: uppercase;
+      text-decoration: none;
+
+      &:hover {
+        color: var(--blue-accent, #fff);
+        text-align: center;
+        font-family: "Roboto", sans-serif;
+        font-size: 14px;
+        font-weight: 700;
+        line-height: 16px;
+        text-transform: uppercase;
```

Comment:
There is a lot of repeating code, consider deleting redundant lines. The element has already inherited styling